packages/player/src/player/basePlayer.ts

```diff
@@ -380,6 +380,11 @@ export class BasePlayer {
   }
 
   finishCurrentMediaProduct(endReason: EndReason) {
+    // A media product was loaded by never started.
```

Comment:
```suggestion
    // A media product was loaded but never started.
```
I think that is what is meant?